openedx/core/djangoapps/user_authn/api/tests/test_views.py

```diff
@@ -175,3 +178,28 @@ def test_user_country_code(self):
 
         assert response.status_code == 200
         assert response.data['countryCode'] == self.country_code
+
+
+@skip_unless_lms
+class SendAccountActivationEmail(APITestCase):
```

Comment:
Inherit with `UserAPITestCase` instead so you don't need to create `APIClient()` by yourself. Also, move things that can be moved to `setUp()` function for better readability:
```
class SendAccountActivationEmail(UserAPITestCase):

    def setUp(self):
        """
        Create a user, then log in.
        """
        super().setUp()
        self.user = UserFactory()
        Registration().register(self.user)
        result = self.client.login(username=self.user.username, password=TEST_PASSWORD)
        assert result, 'Could not log in'
        self.path = reverse('send_account_activation_email')

    @patch('common.djangoapps.student.views.management.compose_activation_email')
    def test_send_email_to_inactive_user_via_cta_dialog(self, email):
        """
        Tests when user clicks on resend activation email on CTA dialog box, system
        sends an activation email to the user.
        """
        self.user.is_active = False
        self.user.save()
        self.client.post(self.path)
        assert email.called is True, 'method should have been called'
```